app/build.gradle

```diff
@@ -3,15 +3,26 @@ apply plugin: 'kotlin-android'
 apply plugin: 'kotlin-android-extensions'
 
 android {
-    compileSdkVersion 28
+    compileSdkVersion 29
 
     defaultConfig {
         applicationId 'com.yubico.yubioath'
         minSdkVersion 15
-        targetSdkVersion 28
+        targetSdkVersion 29
         versionCode 20194
         versionName '2.2.0-rc5'
     }
+    flavorDimensions "openSourcePolicy"
+    productFlavors {
+        // Flavor in which every dependency used should also be open source
+        strict {
+            dimension "openSourcePolicy"
+        }
+        // Relaxed Flavor allows non-open source dependencies to be used
+        relaxed {
```

Comment:
nit: can you name it full instead of relaxed please?

Reply:
But then it would not make much sense with the current dimension name.
This flavor belongs to the `openSourcePolicy` dimension and if we name it `full` it would seem the flavor only has open source code, while it is exactly the opposite! I can change it, provided I also change the dimension name.

What do you suggest for the two flavors and the dimension names?

Reply:
I'd name flavors opensource and full for flavorDimensions "policy".